src/views/TradingCompetition/components/YourScore/CardUserInfo.tsx

```diff
@@ -149,65 +156,75 @@ const CardUserInfo: React.FC<YourScoreProps> = ({ hasRegistered, account, profil
         {subHeadingText}
       </Text>
       {shouldShowUserRanks && (
-        <RanksWrapper>
-          <Flex>
-            {volume > 0 ? (
+        <>
+          {profile.nft && volume > 0 && (
```

Comment:
Only show the button if the profile has an nft attached & the user trading volume is > 0